processing/src/main/java/org/apache/druid/segment/nested/NestedDataColumnSerializer.java

```diff
@@ -398,17 +405,12 @@ void openColumnSerializer(String field, SegmentWriteOutMedium medium, int maxId)
       encodedValueSerializer.open();
     }
 
-    void serializeRow(int globalId, int localId) throws IOException
-    {
-      encodedValueSerializer.addValue(localId);
-    }
-
     long getSerializedColumnSize() throws IOException
     {
       return Integer.BYTES + Integer.BYTES + encodedValueSerializer.getSerializedSize();
     }
 
-    public void open() throws IOException
+    public void open(String field) throws IOException
```

Comment:
This parameter appears to not be getting used?

Reply:
this is getting used by the overrides for long and doubles, https://github.com/apache/druid/pull/13051/files#diff-79a5d0543cecca970d2986e5437ed90ab0964cefc89297a23698de2693e9f9b7R555